python/lsst/ts/authorize/authorize.py

```
@@ -83,19 +181,16 @@ async def do_requestAuthorization(self, data):
             for csc in nonauth_cscs.split(","):
                 utils.check_csc(csc.strip())
 
-        for csc_name_index in cscs_to_command:
-            csc, index = salobj.name_to_name_index(csc_name_index)
-            try:
-                async with salobj.Remote(
-                    domain=self.salinfo.domain, name=csc, index=index, include=[]
-                ) as remote:
-                    await remote.cmd_setAuthList.set_start(
-                        authorizedUsers=data.authorizedUsers,
-                        nonAuthorizedCSCs=data.nonAuthorizedCSCs,
-                        timeout=AUTH_TIMEOUT,
-                    )
-                self.log.info(f"Set authList for {csc_name_index}")
-            except salobj.AckError as e:
-                self.log.warning(
-                    f"Failed to set authList for {csc_name_index}: {e.args[0]}"
-                )
+        # TODO: Implement validation with LOVE system.
+        if not self.config.auto_authorization:
+            self.log.warning("Validating with LOVE not implemented yet. Continuing.")
```

Please consider rejecting the configuration (in the configure method), instead, so that the person running the CSC knows exactly what they are getting. It feels safer than almost-silently failing to ask LOVE for permission after the configuration specifically demands that.

That said, I can see one use case for accepting "true": you could make the default configuration set this "true" so that LOVE starts handling requests as soon as you implement them. I'm uneasy with that, but if that's you preference I won't stand in the way.